src/pages/actions.tsx

```diff
@@ -0,0 +1,129 @@
+import { useAppBridge } from "@saleor/app-sdk/app-bridge";
+import { Button, Text, Box } from "@saleor/macaw-ui/next";
+import { useOrdersQuery } from "../../generated/graphql";
+
+/**
+ * This is example of using AppBridge, when App is mounted in Dashboard
+ * See more about AppBridge possibilities
+ * https://github.com/saleor/saleor-app-sdk/blob/main/docs/app-bridge.md
+ *
+ * -> You can safely remove this file!
+ */
+const ActionsPage = () => {
+  const { appBridge, appBridgeState } = useAppBridge();
+  const [{ data, fetching }] = useOrdersQuery();
+
+  const navigateToOrders = () => {
+    appBridge?.dispatch({
+      type: "redirect",
+      payload: {
+        to: "/orders",
+        actionId: "message-from-app",
+      },
```

Comment:
we can import `Actions.createRedirectAction` (AFAIR) from SDK that will
1. show that app-sdk can do that
2. hide actionId which is implementation detail